### PR TITLE
fix: use virtualenv instead of pipx for renku-python

### PR DESCRIPTION
**Pull request by ableuler**
~Depends on https://github.com/SwissDataScienceCenter/renkulab-docker/pull/187 being released.~ 
Ready to merge.

### CODE REVIEW

File: R-minimal/Dockerfile [hunk 1]
```diff
@@ -35,14 +35,20 @@ ARG RENKU_VERSION={{ __renku_version__ | default("0.16.0") }}
 
 # Install renku from pypi or from github if it's a dev version
 RUN if [ -n "$RENKU_VERSION" ] ; then \
-        currentversion=$(pipx list | sed -n "s/^\s*package\srenku\s\([^,]\+\),.*$/\1/p") ; \
+        source .renku-python/bin/activate ; \
+        version_string=$(pip show renku | grep Version) ; \
+        pattern='Version: (.*)$' ; \
+        [[ $version_string =~ $pattern ]] ; \
+        currentversion=${BASH_REMATCH[1]} ; \
```
Comment (rokroskar) [inline R-minimal/Dockerfile:42]: what's wrong with `renku --version`?

Comment (ableuler) [reply R-minimal/Dockerfile:42]: Can't you see the beauty in that regex...? 🤦 
fair point

Comment (rokroskar) [reply R-minimal/Dockerfile:42]: well maybe @Panaetius had some reason to not use it in the first place either...

Comment (ableuler) [reply R-minimal/Dockerfile]: maybe it simply predates the introduction of `renku --version`?

Comment (rokroskar) [reply R-minimal/Dockerfile]: no that's been around forever

Comment (ableuler) [reply R-minimal/Dockerfile]: Do you know what `renku --version` returns for non-released versions? I guess @Panaetius will tell us :)

Comment (rokroskar) [reply R-minimal/Dockerfile]: something like this: `0.16.1.dev16+g78fad896` - see the long regex at the bottom of the Dockerfile...

Comment (ableuler) [reply R-minimal/Dockerfile]: indeed, `renku --version` and the version displayed by `pip show` or `pipx list` seem to be identical also dev versions. I don't see a reason for not using `renku --version` then.

**Review by Panaetius — APPROVED** [review]
This looks great, thank you!

**Pull request merged**